test/helpers/wallet.ts

```diff
@@ -0,0 +1,11 @@
+import * as Wallets from "@app/wallets"
+import { baseLogger } from "@services/logger"
+
+export const getBTCBalance = async (walletId: WalletId): Promise<Satoshis> => {
```

Comment:
maybe just `getBalance`

Reply:
There's another `getBalance` still hanging around in the code so I was thinking to leave this differentiated for now to make it easier to remove that function later on